src/Microsoft.AspNetCore.Antiforgery/Internal/IAntiforgeryAdditionalDataProvider.cs

```diff
@@ -3,7 +3,7 @@
 
 using Microsoft.AspNetCore.Http;
 
-namespace Microsoft.AspNetCore.Antiforgery
+namespace Microsoft.AspNetCore.Antiforgery.Internal
```

Comment:
Do we want to leave this one public? This is the one documented extensibility point that I'm aware of: http://www.asp.net/mvc/overview/security/xsrfcsrf-prevention-in-aspnet-mvc-and-web-pages#_Scenarios_with_special